Makefile

```diff
@@ -12,32 +12,33 @@
 #See the License for the specific language governing permissions and
 #limitations under the License.
 
-.PHONY: build build-controller build-importer build-cloner build-apiserver build-uploadproxy build-uploadserver build-operator build-functest-file-image-init build-functest-registry-image-init build-functest \
-	    manifests \
+.PHONY: manifests \
 		olm-verify olm-push \
-    	docker docker-controller docker-cloner docker-importer docker-apiserver docker-uploadproxy docker-uploadserver docker-operator docker-functest-image-init docker-functest-image-http docker-functest-registry-populate docker-functest-registry docker-functest-registry-init docker-olm-catalog \
-		cluster-up cluster-down cluster-sync cluster-sync-controller cluster-sync-cloner cluster-sync-importer cluster-sync-apiserver cluster-sync-uploadproxy cluster-sync-uploadserver \
+		cluster-up cluster-down cluster-sync \
 		test test-functional test-unit test-lint \
 		publish \
 		vet \
 		format \
 		goveralls \
-		release-description
+		release-description \
+		bazel-generate bazel-build bazel-build-images bazel-push-images
```

Comment:
not sure `bazel-push-images` should be here.  We didn't push images before when simply running `make`.  This basically requires `make cluster-sync` to have been run first.

Reply:
bazel-push-images builds the images and pushes it to the configured registry. make cluster-sync does that and install CDI as well.

Reply:
Sorry, I meant `make cluster-up`.  Basically, I don't think we should always push to the registry.  Sometimes I just want to make sure everything compiles/builds locally.

Reply:
So those are just entries in the .PHONY section of the makefile, telling it there is no requirement that a file called bazel-generate/etc exist. I am not sure what you mean by make sure everything compiles/builds locally. That is what make bazel-build is for, that builds the binaries.